### DISCUSSION TITLE
I have Tried to preview the Tween Setting inside The Inspector But nothing was Shown

### DISCUSSION THREAD

**Discussion by MohammedMedoukh96** [KyryloKuzyk/PrimeTween#50] [General] [answered]
as Attached Image , I don't know if I missing something or this still did not applied in PrimeTween  
![image](https://github.com/KyryloKuzyk/PrimeTween/assets/90128198/467f3bb1-d8a1-47f0-a6b0-c9cebe482e99)

![image](https://github.com/KyryloKuzyk/PrimeTween/assets/90128198/c36e4faa-b35a-4cb8-a22f-154c451b893c)

**Comment by KyryloKuzyk** [comment, marked as answer]
The generic TweenSettings<T> version can only be serialized and tweaked from the Inspector in Unity 2020.1+ and newer. I should probably mention this in the documentation in addition to the XML documentation:
`Contains 'start' and 'end' values of an animation in addition to TweenSettings. Can be serialized and tweaked from the Inspector in Unity 2020.1+. Use this struct when the 'start' and 'end' values of an animation are known in advance.`

What Unity version are you on? If you're using Unity < 2020.1, then only a non-generic TweenSettings version is available for serialization, which doesn't have the `startValue` and `endValue` fields.

**Reply by MohammedMedoukh96** [reply]
Yes actually I am using Unity 2019, Thanks for quick reply
